lib/tableau_api/connection.rb

```diff
@@ -28,7 +28,7 @@ def api_get_collection(path, collection, *args)
           raise TableauError, res if res.code.to_s != '200'
 
           # ensure the result is an array because it will not be an array if there is only one element
-          [collection.split('.').reduce(res['tsResponse']) { |acc, elem| acc && acc[elem] }].flatten.compact.each do |obj|
+          [collection.split('.').reduce(res['tsResponse']) { |a, e| a && a[e] }].flatten.compact.each do |obj|
```

Comment:
this is required to work on rubocop 0.40.0 but a previous PR changed it so it would work with the latest version of rubocop